recruiter/forms.py

```diff
@@ -33,3 +33,9 @@ class UpdateRecruiterAccountSettingsForm(forms.ModelForm):
     class Meta:
         model = Recruiter
         fields = ["name", "company", "email", "phone_number"]
+
+    # Override the init method to add classes to the form fields
+    def __init__(self, *args, **kwargs):
+        super(UpdateRecruiterAccountSettingsForm, self).__init__(*args, **kwargs)
+        for form_field in self.visible_fields():
+            form_field.field.widget.attrs['class'] = 'form-control'
```

Comment:
Nice, but how would you avoid having to repeat this code in all the forms?

Reply:
I'll create a function that does that and then will call it from each `init` method.
Will push it shortly

Reply:
@ifireball 
Done,
I wasn't sure where to put this function so I put it in `jobboard/forms.py` as `jobboard` is sort of the main app, so it made sense to me.
Please let me know if it makes more sense to put it somewhere else.